scripts/src/chartprreview/verify-report.sh

```diff
@@ -242,21 +254,21 @@ getFails() {
         elif [[ $line == *"outcome:"* ]]; then
            outcome=`echo $line | cut -d: -f2- | xargs`
         elif [[ $line == *"reason:"* ]]; then
-           reason=`echo $line | cut -d: -f2- | xargs`
+           reason=`echo $line | cut -d: -f2-`
            if [[ $reason == *'|-' ]]; then
              multireason=true
              reason=""
            elif [[ $reason == *'|' ]]; then
              nextlinereason=true
              reason=""
+           elif [[ $reason == *\'* ]]; then
+             multireason=true
+             reason=`printf "$reason\n" | sed "s/\"/'/g"`
+             continue
            fi
         elif [ "$multireason" = true ]; then
-          reason=`echo $line | xargs`
-          if [[ $line == *"Image is Red Hat certified"* ]]; then
-            outcome="PASS"
-          else
-            outcome="FAIL"
-          fi
+          reason+=`printf "$line\n" | sed "s/\"/'/g"`
```

Comment:
The two `sed`'s here ensures that no double quotes inside double quotes, which are probamatic for json parser.